usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction_grouped.md

```diff
@@ -0,0 +1,283 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending By Transaction Grouped [/api/v2/search/spending_by_transaction_grouped/]
+
+## POST
+
+Searches for transaction records based on a provided set of filters and groups them by their prime award.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `filters` (required, AdvancedFilterObject)
+            These filters are applied at the Transaction level (`children`). They will not affect the Prime Awards (`results`) themselves. Must at least provide the following in the filters object, `keywords` and `award_type_codes`.
+        + `fields` (required, FieldNameObject)
+            The field names to include for `children` objects (Transactions) the response.
+        + `limit`: 5 (optional, number)
+            The number of `results` (Prime Awards) to include per page. The number of `children` (Transactions) objects will always be limited to 10
+            + Default: 10
+        + `page`: 1 (optional, number)
+            The page of `results` (Prime Awards) to return based on `limit`.
+            + Default: 1
+        + `sort`: `Prime Award ID` (required, string)
+            The field on which to order `results` objects (Prime Awards) in the response. The `children` (Transaction) object's sort will always be `Transaction Amount`
+            + Default: `Prime Award ID`
+        + `order` (optional, enum[string])
+            The direction in which to order `results` (Prime Awards). `asc` for ascending or `desc` for descending. The `children` (Transaction) objects will always be ordered `asc`
+            + Default: `desc`
+            + Members
+                + `asc`
+                + `desc`
+    + Body
+
```

Comment:
Any reason for the two blank lines here? In our codebase, I'm seeing 0-1 blank lines in this section of our contracts.

Reply:
Oh, I see that came from my original contract I wrote for this endpoint. Whoops!